lighthouse-cli/run.js

```diff
@@ -247,9 +226,15 @@ async function runLighthouse(url, flags, config) {
       flags.port = launchedChrome.port;
     }
 
-    const runnerResult = flags.fraggleRock && launchedChrome ?
-       await runLighthouseWithFraggleRock(url, flags, config, launchedChrome) :
-       await lighthouse(url, flags, config);
+    if (flags.fraggleRock) {
+      flags.channel = 'fraggle-rock-cli';
+    } else {
+      flags.channel = 'cli';
```

Comment:
should we emit a console.warning stating that this is deprecated?

Reply:
Yeah but I think it makes more sense in the next PR, once we switch `--fraggle-rock` to `--legacy-navigation`. Right now the default behavior would be to emit the deprecation warning.